trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/getLatestFacilityEndDateValues.test.ts

```diff
@@ -45,24 +45,6 @@ describe('getLatestFacilityEndDateValues', () => {
       // Assert
       expect(result).toEqual({ isUsingFacilityEndDate: true, facilityEndDate: mockFacilityEndDate });
     });
-
-    it('should return the the latest amendment value when the FED has been amended', () => {
```

Comment:
Why has this been removed please? AFAIK, facility end date is unrelated to Estore?

Reply:
I have moved this to `trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/getLatestFacilityEndDateValuest.ff-test.ts` a FF specific test.